dotCMS/src/main/java/com/dotmarketing/cms/content/submit/PluginDeployer.java

```diff
@@ -1,6 +1,7 @@
 package com.dotmarketing.cms.content.submit;
 
 
+@Deprecated
 public class PluginDeployer implements com.dotmarketing.plugin.PluginDeployer {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)